### PR TITLE
[WIP] Feat/extend profile

### PR DESCRIPTION
**Pull request by maurelian**
Extends the contractProfile.js file to include modifiers and storage vars: 

image

I'd love some feedback on the code if you can. Should it be broken up into smaller functions? How do you feel about having a visitor within the visitor?

### CODE REVIEW

**Comment by GNSPS** [comment]
Some changes in the profiler will clash when rebasing (because of @anurag-git 's PR #41, which will be merged sooner), otherwise, this looks good to me. 😄

File: bin/surya [hunk 1]
```diff
@@ -6,12 +6,22 @@ require('yargs') // eslint-disable-line
   .usage('$0 <cmd> [args]')
   .command('describe <files..>', 'show file contracts structure.', (yargs) => {
     yargs
+      .option('m', {
```
Comment (maurelian) [inline bin/surya]: Make printing modifiers and storage vars optional with flags:
`surya describe --modifiers --storage fileName`

**Comment by maurelian** [comment]
OK @GNSPS I think this is ready for review. It only makes changes to the `describe` command. Once I've got your feedback, then we can begin to refactor other commands. 
Let's walk through it sometime in the next days?

**Comment by maurelian** [comment]
Reverting this to WIP. Much work has been done to my profiles branch.

File: src/describe.js [hunk 2]
```diff
@@ -23,11 +22,57 @@ export function describe(files) {
...
+      if(options.storage) {
+        for (let varProfile of profile.stateVarProfiles) {
+          let prefix 
+          
+          if (varProfile.typeInfo.type === 'mapping') {
+            let keyType = varProfile.typeInfo.keyType
+            let valueType = varProfile.typeInfo.valueType
+            
+            if(typeof valueType === 'string') {
+              prefix = `map(${keyType} -> ${valueType}`
```
Comment (GNSPS) [inline src/describe.js]: missing a parentheses

**Comment by maurelian** [comment]
I don't anticipate this PR being merged, but it has changes to the describe command I want to keep handy. So I'm going to keep this PR open to represent WIP on improvements there.

**Pull request closed without merging**